wasm/src/debug_service.rs

```diff
@@ -195,3 +313,43 @@ struct StackFrame {
     pub lo: u32,
     pub hi: u32,
 }
+
+#[wasm_bindgen(typescript_custom_section)]
+const IVariableList: &'static str = r#"
+export interface IVariable {
+    name: string;
+    value: string;
+    var_type: "Array"
```

Comment:
As mentioned in the other comment, it's possible this could just become `string` if you allow the full type to propagate out of the evaluator.

Reply:
We'll take another look at this later. Straightforward solutions slow down execution 2-7%.